redbot/launcher.py

```diff
@@ -16,6 +16,7 @@
     create_backup,
     save_config,
 )
+from redbot.core import __verison__
```

Comment:
Version is misspelled.

Reply:
Fixed. I tested my changes locally, then added the changes manually with github's online editor. I'm going to test my changes in a virtualenv, that'll be better.